tests/tasks/two_moons/test_task.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+import torch
+
+from sbibm.tasks.two_moons.task import TwoMoons
+
+torch.manual_seed(47)
```

Comment:
Could you swap this one for ` pyro.util.set_rng_seed(47)` as well?

Reply:
I could, I would take the liberty to do that everywhere then:
```
$ git --no-pager grep 'torch.manual_seed'
sbibm/tasks/bernoulli_glm/task.py:            torch.manual_seed(observation_seed)
sbibm/tasks/slcp/task.py:        torch.manual_seed(42)
sbibm/tasks/task.py:            torch.manual_seed(observation_seed)
tests/tasks/two_moons/test_task.py:torch.manual_seed(47)
```

Reply:
done [`9ffc0f6` (#29)](https://github.com/sbi-benchmark/sbibm/pull/29/commits/9ffc0f6d7e7e808fac43c067510561d3f0cc1e12)

Reply:
Thanks for checking where else this appears. I would rather restrict the changes to `tests/` in this PR though -- changing the other occurrences would change seeding for regenerating observations which I'd like to avoid

Reply:
:cry: ok, I picked up some git-foo this week with respect to rebasing. Let me see if I can ditch this commit that overdid the pyro-seeding.

Reply:
Should be pretty straighforward -- all you need is `git rebase -i main` (on the feature branch) and then drop the last one :)

Reply:
done with [`abc4a85` (#29)](https://github.com/sbi-benchmark/sbibm/pull/29/commits/abc4a852b00641546cb6591cc72e34ee4da81b61)